### DISCUSSION TITLE
p-table styling is not applying

### DISCUSSION THREAD

**Discussion by workvact** [primefaces/community#153] [PrimeNG] [answered]
I created a stackblitz app using p-table but the table styling doesn't apply. I was originally trying to use bootstrap with PrimeNG and thought it was the bootstrap but removing bootstrap import didn't help. Can you please tell me what I am doing wrong?

https://stackblitz.com/edit/stackblitz-starters-et1vtp

Edit: Updated the link to point to editor.

**Comment by cetincakiroglu** [comment]
Hi,

Your example is broken please update the link so we can investigate the issue

**Reply by workvact** [reply]
Updated the link.

**Reply by cetincakiroglu** [reply]
Hi,

As I see in the example, primeng.min.css and nano theme styles are applied to the table element. What is the issue?

**Reply by workvact** [reply]
Grid lines from this setting is not appearing:
styleClass="p-datatable-gridlines p-datatable-lg"

**Reply by cetincakiroglu** [reply, marked as answer]
You're using styleClass within `ng-template`, that's why it doesn't work. 

Here is the correct usage https://stackblitz.com/edit/stackblitz-starters-sq8d71?file=src%2Fapp.component.html

**Reply by workvact** [reply]
Thank you. I messed up my question. I started with a problem of not seeing grid lines in the project with bootstrap 5. Removed bootstrap in the sample and still didn't see the grid line. Thought it may have nothing to do with bootstrap. Uncommenting the bootstrap import in global_styes.css of the fork caused the grid lines disappear. Should I open another discussion?

Answered here: https://github.com/orgs/primefaces/discussions/178

**Reply by cetincakiroglu** [reply]
Because they are bootstrap classes, you need to import bootstrap CSS to use them.
